backend/internal/sys/config/conf.go

```diff
@@ -32,6 +32,7 @@ type Options struct {
 	AllowRegistration    bool   `yaml:"disable_registration"    conf:"default:true"`
 	AutoIncrementAssetID bool   `yaml:"auto_increment_asset_id" conf:"default:true"`
 	CurrencyConfig       string `yaml:"currencies"`
+	AllowAnalytics       bool   `yaml:"allow_analytics"         conf:"default:true"`
```

Comment:
This needs changing to be default false, this was set to true during testing. This feature will be strictly opt-in only.